PlaygroundGUI/PlaygroundGUI.jucer

```diff
@@ -75,8 +117,8 @@
       </MODULEPATHS>
     </XCODE_MAC>
     <LINUX_MAKE targetFolder="Builds/LinuxMakefile" smallIcon="MDnkQK" bigIcon="MDnkQK"
-                extraCompilerFlags="-I ../../../../automaton/src/local_third_party/include"
-                extraLinkerFlags="../../../../automaton/src/local_third_party/lib/libblockchain_cpp_node.a&#10;../../../../automaton/src/local_third_party/lib/libautomaton-core.a&#10;../../../../automaton/src/local_third_party/lib/libsecp256k1.a&#10;../../../../automaton/src/local_third_party/lib/libboost_filesystem.a&#10;../../../../automaton/src/local_third_party/lib/libboost_iostreams.a&#10;../../../../automaton/src/local_third_party/lib/libboost_system.a&#10;../../../../automaton/src/local_third_party/lib/libssl.a&#10;../../../../automaton/src/local_third_party/lib/libcrypto.a&#10;../../../../automaton/src/local_third_party/lib/libcryptopp.a&#10;../../../../automaton/src/local_third_party/lib/libcurl.a&#10;../../../../automaton/src/local_third_party/lib/libg3logger.a&#10;../../../../automaton/src/local_third_party/lib/libgmp.a&#10;../../../../automaton/src/local_third_party/lib/liblua.a&#10;../../../../automaton/src/local_third_party/lib/libprotobuf-lite.a&#10;../../../../automaton/src/local_third_party/lib/libprotobuf-lite.a&#10;../../../../automaton/src/local_third_party/lib/libprotobuf.a&#10;../../../../automaton/src/local_third_party/lib/libreplxx.a&#10;../../../../automaton/src/local_third_party/lib/libz.a&#10;"
+                extraCompilerFlags="-I ../../../../automaton/src/&#10;-I ../../../../automaton/src/local_third_party/include"

```

Comment:
``-I ../../../../automaton/src/`` should not be needed as automaton/src include files are automatically added to local_third_party/include right now.